parser.go

```diff
@@ -730,6 +730,14 @@ func (v *Value) GetStringBytes(keys ...string) []byte {
 	return s2b(v.s)
 }
 
+func (v *Value) GetString(keys ...string) string {
+	v = v.Get(keys...)
+		if v == nil || v.Type() != TypeString {
+		return ""
+	}
+	return v.s
```

Comment:
This won't work, since `v.s` points to internal byte slice, which is changed on the next `Parse` call. So the returned string will contain garbage after that. Proper `GetString` implementation should look like:
```go
func (v *Value) GetString(keys ...string) string {
    return string(v.GetStringBytes())
}
```

In this case the function will return a copy of `v.s`.